cfpb/design-system-react
# Better align Checkboxes documentation with Design System

Closes #168

## Changes

- Change Storybook name from Checkbox -> Checkboxes
- Language: Standard checkboxs -> Standard checkbox
- Style: Intro Paragraph font to 16px
- Language: Sentence case with all labels (in stories)
- Large target - Remove intro paragraph
- Reorg: Move `Checkbox with helper` down below somewhere

## How to test this PR

1. Visit the [Netlify preview](https://deploy-preview-170--cfpb-design-system-react.netlify.app/?path=/docs/components-checkboxes--overview)
1. Confirm that the page flow feels sensible

## Screenshots
Updated: Sept 29

### Sidebar
![Screenshot 2023-09-29 at 1 06 08 PM](https://github.com/cfpb/design-system-react/assets/2592907/667a8dcd-2220-457b-be53-07a9dddce9f6)

### Overview
![screencapture-localhost-6006-2023-09-29-15_54_15](https://github.com/cfpb/design-system-react/assets/2592907/44b80576-15c0-48c2-b546-220ffe7ad149)
